src/content/en/2021/jamstack.md

```diff
@@ -170,19 +177,23 @@ Subregions give us good enough granularity and some exciting data.
   height=794
 ) }}
 
-It's not a surprise that first in a list stands for Northern America and USA location and driver of technologies. Most companies who invented SSGs are in the USA. But surprisingly, Western Europe is in second place and has 10K SSG mobile sites versus 17k for Northern America, but almost the same adoption level, 1.1%.
+It's no surprise that first in the list is Northern America and USA location and driver of technologies. Most companies who invented SSGs are in the USA. But surprisingly, Western Europe is in second place and has 10k SSG mobile sites versus 17k for Northern America, but almost the same adoption level at 1.1%.
+
+{# TODO - first sentence confuses me #}
 
 The tail also shows great results. Subregions with lower number of sites in general adopt technology even broader, like 4.8% of Micronesia sites.
 
 ### Summary of adoption by regions
 
 SSG technology is adopted not only by places where it's invented and popularized, but also all around the world. Moreover, in places where the next generation of the web will be built, it grows even faster, mostly 2 times and sometimes 4 times, compared to others.
 
+{# TODO - does this need a source? ^ #}
```

Comment:
I think it is sourced in the chart above. But not loving the absolute statement that this is where the next generation is to be built.

To be honest I'd drop this "Summary of adoption by regions" section. Don't think it adds much.

Reply:
Dropped it in latest edit.